cmd/oaas/handler_build.go

```diff
@@ -68,6 +66,7 @@ func runOsbuild(buildDir string, control *controlJSON, output io.Writer) (string
 	cmd.Env = append(cmd.Env, control.Environments...)
 	cmd.Args = append(cmd.Args, []string{"--output-dir", outputDir}...)
 	cmd.Args = append(cmd.Args, []string{"--store", storeDir}...)
+	cmd.Args = append(cmd.Args, "--json")
```

Comment:
Maybe worth adding to "todo" here that we want `--monitor=JSONSeqMonitor` here eventually so that the server can get all the realtime information

Reply:
I can add this later too, probably best to get this merged first.